devo/api/client.py

```diff
@@ -50,41 +49,67 @@
 class DevoClientException(Exception):
     """ Default Devo Client Exception """
 
+    def __init__(self, message, status=None, code=None, cause=None):
```

Comment:
What implications will this change have on the existing code? I guess that's not much, just adding info to the Exception... just asking.